website/pages/style.css

```diff
@@ -1,10 +1,7 @@
 /* Global Styles */
-@import '~@hashicorp/react-global-styles/style.css';
-@import '~@hashicorp/react-global-styles/font.css';
```

Comment:
@jescalan  Already approved since the font looked fine, just curious if `font` is move into the main `style.css`? I can look myself, just something that crossed my mind

Reply:
[Ahh I see it here](https://github.com/hashicorp/mktg-global-styles/blob/main/global.css) 👍

Reply:
weirdly enough, it was actually always in there haha. i just realized it was and removed these